app/move/controllers/view/view.locals.js

```diff
@@ -129,6 +129,7 @@ function getViewLocals(req) {
       ? req.t('statuses::' + status, { context: cancellationReason })
       : undefined,
     messageContent: req.t('statuses::description', {
+      ...move,
```

Comment:
This seems quite overkill. Can we be more specific with what properties the translation string needs rather than passing the whole thing through please?